### PR TITLE
HLT update including Ecal for 75X

### PR DESCRIPTION
**Pull request by Martin-Grunewald**
HLT update including Ecal for 74X: includes #7813 and #7911 and required HLT changes.

### CODE REVIEW

**Comment by Martin-Grunewald** [comment]
please test

**Comment by Martin-Grunewald** [comment]
+1

**Comment by Dr15Jones** [comment]
+1
Just configuration changes

**Comment by diguida** [comment]
+1

**Comment by Martin-Grunewald** [comment]
+1
Rebased to CMSSW_7_5_X_2015-02-25-0200

**Comment by Martin-Grunewald** [comment]
please test

File: HLTrigger/Configuration/python/HLT_GRun_cff.py [hunk 1]
```diff
@@ -1930,73 +1930,67 @@
...
+      ebSpikeThreshold = cms.double( 1.042 ),
+      EBtimeNconst = cms.double( 28.5 ),
+      ampErrorCalculation = cms.bool( False ),
+      kPoorRecoFlagEB = cms.bool( True ),
+      ebPulseShape = cms.vdouble( 5.2E-5, -5.26E-5, 6.66E-5, 0.1168, 0.7575, 1.0, 0.8876, 0.6732, 0.4741, 0.3194 ),
+      EBtimeFitLimits_Lower = cms.double( 0.2 ),
+      kPoorRecoFlagEE = cms.bool( False ),
+      chi2ThreshEB_ = cms.double( 65.0 ),
+      EEtimeFitParameters = cms.vdouble( -2.390548, 3.553628, -17.62341, 67.67538, -133.213, 140.7432, -75.41106, 16.20277 ),
+      useLumiInfoRunHeader = cms.bool( False ),
```
Comment (slava77) [inline HLTrigger/Configuration/python/HLT_GRun_cff.py]: I suppose you will control these manually with specific 25 and 50 ns configurations setting sin activeBXs values
(just checking that the change is intentional)

**Comment by slava77** [comment]
+1

for https://github.com/cms-sw/cmssw/pull/7916	c2c117c
checked locally in CMSSW_7_5_X_2015-02-25-1400 /test area sign521/

I compared expanded py configs. All relevant changes are in process.hltEcalUncalibRecHit and in process.ecalMultiFitUncalibRecHit in standard reco.
- For the standard reco the change is just to move parameters to algoPSet ==> no changes expected in physics outputs (and none observed based on the short matrix test)
- for HLT the changes include those with expected physics changes. There are indeed changes in HLT DQM plots, which are fairly minor with the statistics of the tests used.
```
.. doPrefitEB = cms.bool(True) from False
.. doPrefitEE = cms.bool(True) from False
..  useLumiInfoRunHeader = cms.bool(False)	from True 
	when true would consume 	bunchSpacing_ = c.consumes<int>(edm::InputTag("addPileupInfo","bunchSpacing")
.. prefitMaxChiSqEB = cms.double(100.0) from 25

```

**Comment by Martin-Grunewald** [comment]
@slava77 
Indeed, for 50ns we in fact use the old algorithm:
```
    algo = cms.string( "EcalUncalibRecHitWorkerWeights" ),
    algoPSet = cms.PSet(  )
```

**Comment by davidlange6** [comment]
+1

**Pull request merged**